src/java/org/apache/cassandra/db/commitlog/AbstractCommitLogService.java

```diff
@@ -191,6 +197,9 @@ boolean sync()
                     commitLog.sync(false);
                 }
 
+                if (lastSyncError != null)
```

Comment:
Just assign null? Is there any magic here?

Reply:
I don't know why it was implemented in the first place, but I simplified here accordion your suggestion.